ckanext/nhm/theme/templates/about/citation.html

```diff
@@ -21,23 +21,43 @@ <h1 class="page-heading">{{ title }}</h1>
+
+    <h3>1. Datasets</h3>
+    <p>
+        A single, complete dataset: e.g. the DOI <a href="https://doi.org/10.5519/hay2d5yo">10.5519/hay2d5yo</a> references the <a href="/dataset/join-the-dots-collection-level-descriptions">Join the Dots</a> dataset (including all resources therein).
+    </p>
+    <p>
+        Each dataset overview page will have a 'Cite this as' section with an APA-style reference you can copy, or you can download the BibTeX reference from DataCite using the "BibTeX" button.
+    </p>
+    <div class="citation-string">
+        <u>Author, A., Author, B.</u>. (<u>Year</u>). <u><i>Dataset name</i></u> [Data set]. Natural History Museum. https://doi.org/<u>doi</u>
     </div>
+
+    <h3>2. Resources</h3>
+    <p>We don't currently provide separate DOIs for resources, so if you cite a single resource use the standard URL. As with datasets, each resource overview page has an APA-style example reference that can be copied.</p>
+    <div class="citation-string">
+        <u>Author, A., Author, B.</u>. (<u>Year</u>). <u><i>Resource name</i></u> (from <u><i>Dataset name</i></u>) [Data set]. Natural History Museum. https://data.nhm.ac.uk/dataset/<u>dataset-id</u>/resource/<u>resource-id</u>
+    </div>
+
+    <h3>3. Records</h3>
+    <p>Records are versioned, so you can either cite the most up-to-date version of the record or the version as it was when you initially retrieved it. Both URLs can be found in the 'Cite this as' section at the bottom of a record page.</p>
+    <div class="citation-string">
+        <u>Author, A., Author, B.</u>. (<u>Year</u>). <u><i>Record name</i></u> (from <u><i>Dataset name</i></u>, <u><i>Resource name</i></u>) [Data set record]. Natural History Museum. https://data.nhm.ac.uk/object/<u>record-id</u>/<u>version</u>
+    </div>
+
+    <h3>4. Searches</h3>
+    <p>You can cite results generated from the <a href="{{ h.url_for('search.view') }}">integrated search</a>. Clicking the "Cite" button and entering an email address will generate a DOI for the search. All records within the search are versioned. This allows you to cite subsets of resources, multiple datasets, and exact versions of multiple records.</p>
+    <div class="citation-string">
+        Natural History Museum. (<u>Year</u>). Query on the Natural History Museum Data Portal (data.nhm.ac.uk) (<u>n</u> records) [Data set]. Natural History Museum. https://doi.org/<u>doi</u>
+    </div>
+
+    <h3>5. The Data Portal</h3>
+    <p>Only cite the Portal directly if you are referencing a non-data aspect of the website, e.g. discussing the concept in general.</p>
+    <p>Implementations of specific features can be found on our <a href="https://github.com/NaturalHistoryMuseum">GitHub</a> repositories, so if you wish to reference code then it may be better to target these specifically.</p>
```

Comment:
It's a shame we can't link the private data-portal repo up to zenodo cause then we could get DOIs for each version we release. Dang private repo 🙁